lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -64,10 +64,12 @@ def __init__(
             The parent figure.
         rect : (float, float, float, float)
             The ``(left, bottom, width, height)`` axes position.
-        azim : float, default: -60
-            Azimuthal viewing angle.
         elev : float, default: 30
             Elevation viewing angle.
+        azim : float, default: -60
```

Comment:
Can we briefly define these angles here as in `view_init`?

Reply:
Done